tap_github/schemas/issue_events.json

```diff
@@ -1,1473 +1,962 @@
-            "received_events_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "subscriptions_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "gists_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "html_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "repos_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "events_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "login": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "followers_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "starred_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "avatar_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "id": {
-              "type": [
-                "null",
-                "integer"
-              ]
-            },
-            "type": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "site_admin": {
-              "type": [
-                "null",
-                "boolean"
-              ]
-            },
-            "node_id": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "organizations_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "following_url": {
-              "type": [
-                "null",
-                "string"
-              ]
-            }
-          }
+          "$ref": "shared/user.json#/"
         },
         "comments": {
-          "type": [
-            "null",
-            "integer"
-          ]
+          "type": ["null", "integer"]
         },
         "author_association": {
-          "type": [
-            "null",
-            "string"
-          ]
+          "type": ["null", "string"]
         },
         "milestone": {
-          "type": [
-            "null",
-            "object"
-          ],
+          "type": ["null", "object"],
           "properties": {
             "closed_at": {
-              "type": [
-                "null",
-                "string"
```

Comment:
As per the API doc, the format is date-time for closed_at, please add the same

Reply:
Updated the field to keep the date-time format